metrics/firebase/test/firestore/rules/test_utils/test-data.js

```diff
@@ -182,4 +206,4 @@ exports.getUserProfile = function () {
 /** Get a test task */
 exports.getTask = function () {
   return cloneDeep(tasks["tasks/1"]);
-};
+};
```

Comment:
```suggestion
};

```
![image](https://user-images.githubusercontent.com/442086/130431842-88dc25aa-ab20-416d-8159-fd232165b04f.png)